docs/containers.conf.5.md

```diff
@@ -11,7 +11,9 @@ a TOML format that can be easily modified and versioned.
 
 Container engines read the __/usr/share/containers/containers.conf__,
 __/etc/containers/containers.conf__, and __/etc/containers/containers.conf.d/\*.conf__
```

Comment:
If you have to repush, mind squashing the trailing whitespace here?

Reply:
I don't see any trailing whitespace?

Reply:
Ah, they're literal underscores. Nevermind.